tests/conftest.py

```diff
@@ -0,0 +1,104 @@
+import time
+
+import apypie
+import pytest
+
+FOREMAN_PORT=3000
+
+def to_native(thing):
+    return thing
+
+class ForemanApi(apypie.Api):
```

Comment:
ignore this class for now, lots of :spaghetti: from foreman-ansible-modules

Reply:
It'd be interesting to see what we could add to apypie to make it easier to consume.

Reply:
that's on my plate for next week :)

Reply:
https://github.com/theforeman/foreman-ansible-modules/pull/1789 has a reasonable first stab at that